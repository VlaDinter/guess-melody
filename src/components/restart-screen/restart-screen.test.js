import React from 'react';
import renderer from 'react-test-renderer';

import RestartScreen from './restart-screen.jsx';

it(`RestartScreen is rendered correctly`, () => {
  const tree = renderer.create(<RestartScreen
    onRestartGame={jest.fn()}
  />, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
