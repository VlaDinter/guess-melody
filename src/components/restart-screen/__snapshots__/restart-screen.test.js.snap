// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestartScreen is rendered correctly 1`] = `
<section
  className="modal"
>
  <button
    className="modal__close"
    type="button"
  >
    <span
      className="visually-hidden"
    >
      Закрыть
    </span>
  </button>
  <h2
    className="modal__title"
  >
    Подтверждение
  </h2>
  <p
    className="modal__text"
  >
    Вы уверены что хотите начать игру заново?
  </p>
  <div
    className="modal__buttons"
  >
    <button
      className="modal__button button"
      onClick={[MockFunction]}
    >
      Ок
    </button>
    <button
      className="modal__button button"
    >
      Отмена
    </button>
  </div>
</section>
`;
